### DISCUSSION TITLE
CGNS to SU2 format

### DISCUSSION THREAD

**Discussion by Rteg54** [su2code/SU2#2384] [Q&A]
Hi Community,
i have  done a mesh using  ICEM Ansys, and i have  the  mesh file as cgns now and i want  to change  it to su2 format  and i wrote  a converting cfg  file for thatbut  when i try to run the simulation using su2 code it  says error.
The case i am working with is  a concentric cylinder coutte  flow 3d. As i have  basically a hollow cylinder where the inner cylinder rotates and the  outer  cylinder is fixed the way i named my boundary conditions in ICEM is inner_wall for the inner cylinder then outer_wall for outer cylinder then front is inlet and back is outlet. then i wrote the convert fill as the following:
% SET BOUNDARY MARKERS ACCORDING TO YOUR ICEMCFD PROJECT
MARKER_INLET= ( "INLET" )              % Inlet boundary
MARKER_OUTLET= ( "OUTLET" )            % Outlet boundary
MARKER_WALL_FUNCTIONS= ( "OUTER_WALL", "INNER_WALL" ) % Viscous walls

% MESH INPUT FILE
MESH_FILENAME= T1.cgns                 % Input CGNS file

% MESH INPUT FILE FORMAT (SU2, CGNS, NETCDF_ASCII)
MESH_FORMAT= CGNS                      % Format of the input mesh

% MESH OUTPUT FILE
MESH_OUT_FILENAME= T1.su2              % Output SU2 file

% SOME DUMMY PARAMETERS
DV_KIND= NO_DEFORMATION
DV_PARAM= ( 1.0 )
i would really appreciate the help.

**Comment by bigfooted** [comment]
Was there an error message? And you used SU2_DEF for the conversion?
Does the cgns work directly with SU2?
